test/PhpParser/Node/Scalar/DNumberTest.php

```diff
@@ -18,7 +18,7 @@ public function testRawValue(): void {
         $lLumber = $echo->exprs[0];
```

Comment:
```suggestion
        $dnumber = $echo->exprs[0];
```
We should change the variable instead.